Gemfile

```diff
@@ -43,15 +43,20 @@ group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
+  gem 'database_cleaner', '~> 1.6.1'
```

Comment:
Database cleaner isn't configured, maybe it's in another PR? if that's the case i think we will need to checkout that file from that branch

Reply:
It should be here